cpp/tests/io/parquet_test.cpp

```diff
@@ -1483,5 +1483,120 @@ TEST_F(ParquetReaderTest, ReorderedColumns)
     cudf::test::expect_columns_equal(result.tbl->view().column(3), a);
   }
 }
+/*
+MATCHER_P(FloatNearPointwise, tolerance, "Out-of-range")
+{
+  return (std::get<0>(arg) > std::get<1>(arg) - tolerance &&
+          std::get<0>(arg) < std::get<1>(arg) + tolerance);
+}
+
+template <typename T>
+using wrapper = cudf::test::fixed_width_column_wrapper<T>;
+
+// temporary method to verify the float columns until
+// CUDF_TEST_EXPECT_COLUMNS_EQUAL supports floating point
+template <typename T, typename valid_t>
+void check_float_column(cudf::column_view const& col_lhs,
+                        cudf::column_view const& col_rhs,
+                        T tol,
+                        valid_t const& validity)
+{
+  auto h_data = cudf::test::to_host<T>(col_rhs).first;
+
+  std::vector<T> data(h_data.size());
+  std::copy(h_data.begin(), h_data.end(), data.begin());
+
+  CUDF_TEST_EXPECT_COLUMN_PROPERTIES_EQUIVALENT(col_lhs,
+                                                (wrapper<T>{data.begin(), data.end(), validity}));
+  EXPECT_THAT(cudf::test::to_host<T>(col_lhs).first,
+              ::testing::Pointwise(FloatNearPointwise(tol), data));
+}*/
 
+TEST_F(ParquetReaderTest, DecimalRead)
+{
+  std::string filepath = "cpp/tests/io/data/decimals.parquet";
```

Comment:
The relative path here doesn't seem to work in jenkins pipeline.
```
unknown file
C++ exception with description "cuDF failure at: /var/lib/jenkins/workspace/rapidsai/gpuci-a100/cudf/prb/cudf-gpu-build_2/cpp/src/io/utilities/datasource.cpp:57: Cannot open file" thrown in the test body.
```

Reply:
Yes, I just pushed a fix for that. After talking it over with @harrism I decided to embed the file since it is so small and we don't want to encourage datasets in cudf.